README.md

```diff
@@ -13,19 +13,19 @@ dependencies: ["https://github.com/DataDog/serverless-plugin-datadog/blob/master
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue)](https://github.com/DataDog/serverless-plugin-datadog/blob/master/LICENSE)
```

Comment:
Just be mindful that if you include the README change in the same PR as the code change (which I think is fine in this case), you're going to want to release a new version of the plugin immediately after you merge the PR. Otherwise the docs will update but the latest released version of the plugin won't have the new functionality.